src/sentry/discover/arithmetic.py

```diff
@@ -266,6 +268,8 @@ def parse_arithmetic(
     result = visitor.visit(tree)
     if len(visitor.fields) > 0 and len(visitor.functions) > 0:
         raise ArithmeticValidationError("Cannot mix functions and fields in arithmetic")
+    if visitor.terms == 1:
```

Comment:
Can they somehow send an arithmetic expression with 0 terms? Maybe change this to `visitor.terms <= 1` to be safe.

Reply:
They can't afaik (`+` or `    ` isn't valid to the parser), but no reason not to be safe :+1: